concent_api/middleman/utils.py

```diff
@@ -43,22 +43,22 @@ def update(self, mapping: Dict[int, asyncio.Queue], **kwargs: Any) -> None:  # t
     def pop(self, key: int, **kwargs: Any) -> asyncio.Queue:  # type: ignore
```

Comment:
Shouldn't there be also `*args` here ? I saw them in one of the temporary fixes.

Reply:
good catch:)
done